### ISSUE TITLE
Parser factory is unused

### ISSUE THREAD

**Issue by kelunik** [clue/php-redis-server#15]
The parser factory is unused, because `$parser` is immediately replaced.

https://github.com/clue/php-redis-server/blob/bfe79beadfe76b9d532d766a732b43104e6587c9/src/Server.php#L76-L77

**Comment by clue** [comment]
Indeed, thank you for spotting! Does anybody feel like filing a PR? :+1:
